recipes/bioconda-utils/meta.yaml

```diff
@@ -78,7 +78,7 @@ test:
       --file="$( python -c
       'from os.path import dirname, join; import bioconda_utils; print(join(dirname(bioconda_utils.__file__), "bioconda_utils-requirements.txt"))'
       )"
-      --json | grep -qF 'All requested packages already installed.'
+      --json | tee /dev/stderr | grep -qF 'All requested packages already installed.'
```

Comment:
IDK whether `tee /dev/stderr` works on macOS or not. We'll see what the CI says..